python/simple/v1/ampli/ampli.py

```diff
@@ -0,0 +1,818 @@
+# 
+# ampli.py
+# 
+# Ampli - A strong typed wrapper for your Analytics
+# 
+# This file is generated by Amplitude.
+# To update run 'ampli pull python-ampli'
+# 
+# Required dependencies: amplitude-analytics
+# Tracking Plan Version: 0
+# Build: 0.0.0
+# Runtime: python-ampli
+# 
+# [View Tracking Plan](https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/latest)
+# 
+# [Full Setup Instructions](https://data.amplitude.com/test-codegen/Test%20Codegen/implementation/main/latest/getting-started/python-ampli)
+# 
+# pylint: skip-file
+# flake8: noqa
+
+
+import logging
+import enum
+from typing import Dict, Optional, List, Any, Union
+
+from amplitude import *
+
+
+class Environment(enum.Enum):
+    PRODUCTION = 'production'
+    DEVELOPMENT = 'development'
+    
+
+class LoadClientOptions:
+    
+    def __init__(self, api_key: Optional[str] = None,
+                 instance: Optional[Amplitude] = None,
+                 configuration: Optional[Config] = None):
+        self.api_key = api_key
+        self.instance = instance
+        self.configuration = configuration
+
+
+class LoadOptions:
+
+    def __init__(self, environment: Environment = None,
+                 disabled: bool = False,
+                 client: LoadClientOptions = None):
+        self.environment = environment
+        self.disabled = disabled
+        self.client = client
+        
+        
+class IdentifyProperties:
```

Comment:
Can we have Identify/Group classes extend BaseEvent like the other Events.

Reply:
Core SDK expose Identify/IdentifyEvent/GroupIdentifyEvent classes. For here should I not import these classes from core SDK and implement Identify/Group class extend BaseEvent? Identify class from core SDK is used by Ampli identify/group_identify method now. Or I can add AmpliIdentifyEvent extend IdentifyEvent and AmpliGroupIdentifyEvent extend GroupIdentifyEvent?